samples/drivers/led_max7219/src/main.c

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2022 Jimmy Ou <yanagiis@gmail.com>
+ * Copyright (c) 2018 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/device.h>
+#include <zephyr/drivers/led.h>
+#include <zephyr/sys/util.h>
+#include <zephyr/zephyr.h>
+
+#define LOG_LEVEL CONFIG_LOG_DEFAULT_LEVEL
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(app);
+
+#define NUM_LEDS 64
+
+#define DELAY_TIME K_MSEC(1000)
+
+void main(void)
+{
+	const struct device *led_dev = DEVICE_DT_GET_ANY(maxim_max7219);
+	int i, ret;
+
+	if (led_dev == NULL) {
+		LOG_ERR("No device with compatible maxim,max7219 found");
+		return;
+	}
```

Comment:
why not `DEVICE_DT_GET_ONE`?

Reply:
It's better than `DEVICE_DT_GET_ANY`. Thanks for your suggestion. I will change it.